lib/chef/provider/package/homebrew.rb

```diff
@@ -83,9 +86,34 @@ def brew(*args)
         # information, but that is not any more robust than using the
         # command-line interface that returns the same thing.
         #
-        # https://github.com/Homebrew/homebrew/wiki/Querying-Brew
+        # https://docs.brew.sh/Querying-Brew
+        #
+        # @returns [Hash] a hash of package information where the key is the package name
         def brew_info
-          @brew_info ||= Chef::JSONCompat.from_json(brew("info", "--json=v1", new_resource.package_name)).first
+          @brew_info ||= begin
+            command_array = ["info", "--json=v1"].concat Array(new_resource.package_name)
```

Comment:
```suggestion
            command_array = ["info", "--json=v1", new_resource.package_name]
```

Should be the same, unless I'm misunderstanding what this is doing...

Reply:
We concat here because with multipackage it may be an array and this way we avoid needed to flatten it later:

```ruby
irb(main):001:0> command_array = ["info", "--json=v1"].concat Array("foo")
irb(main):002:0> command_array
=> ["info", "--json=v1", "foo"]
irb(main):003:0> command_array = ["info", "--json=v1"].concat Array(%w(foo))
irb(main):004:0> command_array
=> ["info", "--json=v1", "foo"]
irb(main):005:0> command_array = ["info", "--json=v1"].concat Array(%w(foo bar))
irb(main):006:0> command_array
=> ["info", "--json=v1", "foo", "bar"]
irb(main):007:0> name = "foo"
irb(main):008:0> command_array = ["info", "--json=v1", name]
irb(main):009:0> command_array
=> ["info", "--json=v1", "foo"]
irb(main):010:0> name = %w(foo bar)
irb(main):011:0> command_array = ["info", "--json=v1", name]
irb(main):012:0> command_array
=> ["info", "--json=v1", ["foo", "bar"]]
irb(main):013:0>
```

Reply:
Ahh, that makes sense. 👍